jira.md

```diff
@@ -0,0 +1,11 @@
+## JIRA
+
+Guidelines and rules for managing and working on JIRA. Rules are *obligatory*, suggestions are suggestions.
+
+### Ticket
+
+_Rules:_
+1. Help your TL/PM a bit and manage ticket status by yourself. For example, if a feature goes through
```

Comment:
How does this rule correspond to the Kanban flow we are introducing in some projects?

Reply:
I think this is orthogonal to Kanban. It simply means that a person responsible for a ticket should move it from one column to the next.